scout/server/blueprints/variant/templates/variant/utils.html

```diff
@@ -22,10 +22,7 @@ <h5 class="modal-title" id="causativeLabel">Mark variant as causative</h5>
                     <br>
                     <div>
                       <input type="checkbox" class="ios8-switch" id="partial_causative" name="partial_causative"
-                        data-toggle='collapse' data-target='#partial_phenotypes'
-                          {% if case.phenotype_terms|length == 0 and case.diagnosis_phenotypes|length == 0 %}
```

Comment:
This is the original problem of the bug (disabled checkbox to assign variant as partial causative)